ivy/functional/backends/tensorflow/creation.py

```diff
@@ -285,6 +285,8 @@ def linspace(
     if axis is None:
         axis = -1
     with tf.device(device):
+        start = tf.cast(start, dtype=dtype)
+        stop = tf.cast(stop, dtype=dtype)
         start = tf.cast(tf.constant(start), dtype=dtype)
         stop = tf.cast(tf.constant(stop), dtype=dtype)
```

Comment:
I had already fixed this in the two lines below yours 😅 If the test passes with your fix, we should keep it instead of mine as it's simpler. If so, please commit it directly to master so we can keep this PR focused on pad.

Reply:
Sure, I'll commit this to the master 👍